### DISCUSSION TITLE
Whenever I run atri start command on my CMD is show following error

### DISCUSSION THREAD

**Discussion by Sooraj2002** [Atri-Labs/atrilabs-engine#639] [Help: Installation & Start] [answered]
![atri12](https://user-images.githubusercontent.com/72682874/209805756-ed0aff80-8b22-4981-927a-814faf15fc1f.JPG)
![atri11](https://user-images.githubusercontent.com/72682874/209805760-44ab35a4-fdcf-48cf-862a-cca88fbb4dd0.JPG)
Please resolve my issue so I can use atri framework on my system.

**Comment by cruxcode** [comment, marked as answer]
Please use command `pipenv install atri` instead.

**Comment by ramchaudhary2000** [comment]
Same issue here
pipenv is not recognized as an internal or external command, operable
program or batch file.

**Reply by cruxcode** [reply]
@ramchaudhary2000 your issue is not the same as posted in this discussion.
1. Make sure you have python >=3.8 & <=3.10.x installed. 
2. Install pipenv by running `python -m pip install pipenv`.
3. Make sure Python Scripts is in the PATH environment variable. Instructions [here](https://github.com/Atri-Labs/atrilabs-engine/discussions/586). 
4. Open a new terminal to use `pipenv` related commands.

**Comment by DivyMani01** [comment]
@ramchaudhary2000 solution Stape:-

1.)open Your Windows (Type here to Search). 
2.)Search (turn windows features on or off) Enter. 
3.) ☑ (Windows Subsystem for Linux.)then OK . 
4.) Restart. 
5.)Go-to Microsoft Store. 
6.)Download Ubuntu App. & Open Ubuntu Terminal. 
7.)Enter User name, Password; 
8.)Command (sudo apt update). 
9.)sudo apt install pip 
10.)sudo apt install pipenv 
11.)pipenv shell
12.)pipenv install atri
13.)mkdir first_app
14.)cd first_app
15.)atri start
